lib/index.js

```diff
@@ -372,6 +394,28 @@ function opt (option) {
   }
 };
 
+/* chan and rmchan sockopt methods. only relevant for subscription sockets */
+Socket.prototype.chan       = function(){
+  var args = arguments;
+  var channels = args.length;
+  while(channels--) {
```

Comment:
Would prefer that variables used for the sole purpose of iteration be named `i`, `j`, or `k`.